packages/saber/lib/utils/assetsAttribute.js

```diff
@@ -0,0 +1,44 @@
+// @ts-check
+
+/**
+ * Check if it's external resource
+ * @param {string} str
+ */
+const isExternal = str => /^https?:\/\//.test(str) || /^\//.test(str)
+
+const MARK = '@@!!SABER_ASSET_MARK_e5968b9a!!@@'
+
+const MARK_GLOBAL_RE = new RegExp(`"${MARK}([^"]+)"`, 'g')
+
+/**
+ * Prefix MARK to asset path
+ * @param {{[key: string]: string}} assets
+ */
+const prefixAssets = assets => {
+  /** @type {{[key: string]: string}} */
+  const result = {}
+  for (const key of Object.keys(assets)) {
+    const value = assets[key]
+    if (value.startsWith(MARK)) {
+      result[key] = value
+    } else if (!isExternal(value)) {
+      result[key] = `${MARK}${value}`
+    }
+  }
+
+  return result
+}
+
+/**
+ * Replace strings starting with the MARK to `require` call
+ * @param {string} str
+ */
+const markToRequire = str =>
```

Comment:
I think `markToRequire` is a description that fits to both functions, I'd say it even fits `prefixAssets` better. Would prefer something like `requireAssets` or `injectAssets`